### PR TITLE
add db seed-data check on backend startup

### LINKED ISSUE #133: Insert two dummy conversations in an empty database
Determine if posts table is empty upon backend startup. If so, insert 4 dummy posts to emulate two mini-conversations, e.g the two examples from the [Prompting Guide](https://github.com/LAION-AI/Open-Assistant/blob/main/Prompting_Guide.md). This allows a smoother operation of the `/tasks/` REST endpoint  (e.g. when a random task is requested) in the very early phase after deployment and during development.

It could be placed in [backend/main.py](https://github.com/LAION-AI/Open-Assistant/blob/main/backend/main.py) similar to the alembic update.

### PR DESCRIPTION
**Pull request by andreaskoepf**
This inserts dummy conversations into the posts table if they are not present during startup. This behavior can be disabled by setting the `USE_SEED_DATA` config param to `False`.

Resolves #133 

(Turned out to be more struggle than I initially thought with all the ids. Also good way to review current API maybe.)

### CODE REVIEW

File: backend/oasst_backend/config.py [hunk 1]
```diff
@@ -33,6 +33,7 @@ def assemble_db_connection(cls, v: Optional[str], values: Dict[str, Any]) -> Any
 
     BACKEND_CORS_ORIGINS: List[AnyHttpUrl] = []
     UPDATE_ALEMBIC: bool = True
+    USE_SEED_DATA: bool = True
```
Comment (yk) [inline backend/oasst_backend/config.py:36]: could we set this to false, but set to true in the docker-compose?

**Review by yk — APPROVED** [review]

File: backend/oasst_backend/config.py (hunk 1 above)
Comment (andreaskoepf) [reply backend/oasst_backend/config.py:36]: good idea, will do.

Comment (andreaskoepf) [reply backend/oasst_backend/config.py]: yes, is done.

**Comment by AbdBarho** [comment]
Thank you ❤, this was blocking us on the frontend!

**Pull request merged**